app/models/miq_request_workflow.rb

```diff
@@ -1507,9 +1507,9 @@ def ws_schedule_fields(values, _fields, data)
   def raise_validate_errors
     errors = []
     fields { |_fn, f, _dn, _d| errors << f[:error] unless f[:error].nil? }
-    err_text = "Provision failed for the following reasons:\n#{errors.join("\n")}"
-    _log.error("<#{err_text}>")
-    raise _("Provision failed for the following reasons:\n%{errors}") % {:errors => errors.join("\n")}
+    err_text = errors.join("\n").insert(0, "\n")
```

Comment:
Note, the newline might cause issues if translators don't embed it in their translations.

See: https://github.com/ManageIQ/manageiq/pull/23218#issuecomment-2386566380